client/ayon_core/plugins/publish/collect_aovs_to_anatomy_instance_data.py

```diff
@@ -0,0 +1,22 @@
+import pyblish.api
+
+
+class CollectAOVNameToAnatomyInstanceData(pyblish.api.InstancePlugin):
+    """
+    Collect AOV name to template data.
+
+    Note:
+        This functionality depends on AOV name added by
+        ``submit_publish_job`` plugin and so it only works
+        with Deadline submission.
+
+    """
+    order = pyblish.api.CollectorOrder + 0.499
+    targets = ["farm"]
+    label = "Collect AOV name to template data"
+
+    def process(self, instance):
+        if "aovName" not in instance.data:
+            return
+
+        instance.data["anatomyData"]["aov_name"] = instance.data["aovName"]
```

Comment:
I'm a bit worried about this directly injecting into `anatomyData` - shouldn't that be way more 'static' or 'constant'`? - Why do we need it in anatomy data?

Looking at the overal logic in `pyblish_functions.py` each AOV is already gets separated into its own instance, and thus included in the 'variant' for the instance and thus the subset name. Now also adding a separate `{aov_name}` in there just seems like duplicating that?

The only reason I can think of requiring this `{aov_name}` then is to publish multiple AOVs within a single instance? But we don't seem to ever be doing that.